tests/test-cases/demo-stacks.yaml

```
@@ -150,7 +150,7 @@ tests:
         - "^$"
       exit_code: 0
   - name: atmos_vendor_pull
-    enabled: true
+    enabled: false
```

No, don't disable all tests for vendoring. Disable specifically the test for OCI. 

- https://github.com/cloudposse/atmos/blob/141c7312dc48a1f2fcf715c4625c1c3298856ebd/tests/fixtures/scenarios/complete/vendor.yaml#L28
- https://github.com/cloudposse/atmos/blob/141c7312dc48a1f2fcf715c4625c1c3298856ebd/tests/fixtures/scenarios/complete/components/terraform/infra/vpc2/component.yaml#L18

---

This was fixed in #1093, so we should re-enable OCI vendoring
```suggestion
    enabled: true
```